src/behavior/BehaviorInternalsImpl.ts

```diff
@@ -33,6 +33,7 @@ import BehaviorFlags from "behavior/BehaviorFlags";
 
 const CHANNEL_NAME: string = "channelName";
 const MSG_NAME: string = "messageName";
+const EVENT_NAME: string = "cydran:behavior";
```

Comment:
Let's move this into HardValues so that it can be sued in Behavior.ts as well

Reply:
This value is exclusive to this class and never should be exposed for general use, so it isn't something that we want to advertise externally.

Reply:
There is another class that uses it there in your commit, otherwise I would agree with you.

Reply:
This is now isolated to this class alone.